example/spec/ingestGranule/IngestGranule.output.payload.template.json

```diff
@@ -3,22 +3,33 @@
   "granules": [
     {
       "granuleId": "MOD09GQ.A2016358.h13v04.006.2016360104606",
+      "published": true,
+      "cmrLink": "https://cmr.uat.earthdata.nasa.gov/search/granules.json?concept_id=G1222231611-CUMULUS",
       "files": [
         {
           "name": "MOD09GQ.A2016358.h13v04.006.2016360104606.hdf",
-          "bucket": "",
-          "filename": "",
+          "bucket": "cumulus-test-sandbox-protected",
+          "filename": "s3://cumulus-test-sandbox-protected/MOD09GQ/2017/MOD/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf",
           "fileSize": 17865615,
           "path": "cumulus-test-data/pdrs",
-          "url_path": "{cmrMetadata.Granule.Collection.ShortName}/{extractYear(cmrMetadata.Granule.Temporal.RangeDateTime.BeginningDateTime)}/{substring(file.name, 0, 3)}"
+          "url_path": "{cmrMetadata.Granule.Collection.ShortName}/{extractYear(cmrMetadata.Granule.Temporal.RangeDateTime.BeginningDateTime)}/{substring(file.name, 0, 3)}",
+          "filepath": "MOD09GQ/2017/MOD/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf"
```

Comment:
Why do we need filepath?  Does the s3 filename contain path already?

Reply:
Seems possibly useful.  All of the AWS APIs use  `{ "Bucket": "...", "Key": "..." }`.  Judgment call as to which is more useful to operators, but please do ensure whatever we're calling these things are consistent among tasks we build.